docs/runtime/core.md

```diff
@@ -7,72 +7,72 @@ Runtime Core is a compilation/execution engine for neural network models.
 Runtime Core has four modules. These are namespaces as well as directory names in `/runtime/onert/core/src/`.
 
 - `ir`  stands for Intermediate Representation which contains Neural Network Graph data structures
-- `compiler` converts IR to executable format
-- `exec` is execution module which is the result of compilation
-- `backend` is backend interface
+- `compiler` converts IR to an executable format
+- `exec` is an execution module which is the result of a compilation
+- `backend` is a backend interface
 
 ### Module `ir`
 
 This module contains data structures of pure Neural Network models. The models from NN Packages or NN API are converted to these structures.
 
 - `Subgraphs` is the entire neural network model which is a set of subgraphs
 - `Subgraph` consists of operands and operations
-- `Operand` (a.k.a. Tensor) has shape, data type, data and references to operations
-- `Operation` (a.k.a. Operator) has operation type, params and references to operands
+- `Operand` (a.k.a. Tensor) has a shape, data type, data and references to operations
+- `Operation` (a.k.a. Operator) has operation type, params, and references to operands
 
 `Operand` and `Operation` are nodes of the graph, and the reference relationship between them is the edges of the graph.
 
-`Subgraphs` represents the whole model. It could have more than one `Subgraph` to support control flow operations. Those operations can make calls to another subgraph and when the execution on another subgraph is done it gets back to previous subgraph execution with returned operands.
+`Subgraphs` represents the whole model. A model can have more than one `Subgraph` to support control flow operations. Those operations make calls to another subgraph and when the execution on another subgraph is done it gets back to previous subgraph execution with returned operands.
 
 All graphs are a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) so once model inputs are given we can run it in topological order.
 
-Here's a figure of how those data structures are oraganized.
+Here's a figure of how those data structures are organized.
 
 ![Core](core-figure-ir.png)
 
 ### Module `compiler`
 
 `Compiler` is the main class of this module. Everything starts from it.
 
-What it does is making models executable. It schedules run order and assigns a backend for each operation. Here are major phases of compilation.
+What it does is making models executable. It schedules execution order and assigns a backend for each operation. Here are major phases of compilation.
 
 #### 1. Lowering
 
-In Lowering, the compiler assigns a [backend](#) for each operation. A backend is assigned to an operation means that the operation will be run with the assigned backend's kernel.
+In Lowering, `Compiler` assigns a [backend](#) for each operation. It means that the operation will be run with the assigned backend's kernel.
```

Comment:
@ragmani From #2607 

And I replaced the whole subject with pronoun 'it' since it looks redundant 😄